packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/PeoplePanel/ContactCard/FormRoleSelect.tsx

```diff
@@ -53,18 +51,18 @@ export const FormRoleSelect = ({
           ]}
           formId={formId}
           placeholder='Role'
-          chakraStyles={{
+          styles={{
             multiValue: (props, data) => {
               const colorScheme = (() => getTagColorScheme(data.data.label))();
 
               return {
                 ...props,
                 fontSize: 'xs',
                 fontWeight: 'normal',
-                color: `${[colorScheme]}.700`,
+                color: colors[colorScheme][700],
                 border: '1px solid',
-                borderColor: `${[colorScheme]}.200`,
-                backgroundColor: `${[colorScheme]}.50`,
+                borderColor: colors[colorScheme][200],
+                backgroundColor: colors[colorScheme][50],
 
```

Comment:
Colors should not be passed like this. Either use `classNames` property from Select to pass tw classes and overwrite styles or at least use css variables. but do not import the colors object here. That is meant to go away at compile time.